lib/src/generator/templates.runtime_renderers.dart

```diff
@@ -14757,25 +14757,25 @@ const _invisibleGetters = {
     'unnamedConstructor'
   },
   'CommentReferable': {
+    'packageGraph',
```

Comment:
:( At some point I tried to make this file stable, but it appears there is definitely still instability.

Reply:
Ah.  I didn't look closely but indeed there is.... no worries, it's hard to nail this down.  We'll get there.